### ISSUE TITLE
[AMF]: MobilityD IPv6 block pool initialization issue

### ISSUE THREAD

**Issue by panyogesh** [magma/magma#12068]
As per the Spec : 3GPP TS 24.501 version 15.6.0 Release 15 : 9.11.4.10 PDU address

While sending the IPv6 address to the UE, AMF can only send the interface identifier (last 64 bits). 
UE needs to assume the first 64 bits as per Link local Unicast (fe80::/64) and send accordingly the data traffic. 

With MobilityD currently following behavior is observed :
1. Currently the max prefix len allowed is 58
2. Currently the value of session part bits from rest of 64 prefix bits cannot  be 0

In summary fe80:: + Interface ID is not getting configured in pipelined hence the traffic cases are failing. 

Any reason for the specific behavior.

**Comment by pshelar** [comment]
The max prefix length (58) is the limit of smallest IPv6 IP address block that can is assigned the the pool.
MME does send the prefix as zeros. @panyogesh Are you still seeing the issue?

**Comment by panyogesh** [comment]
Will check and update.
